### DISCUSSION TITLE
How can i run llamaindex with llamafile, is the llamafile embedding support there in the ts version?

### DISCUSSION THREAD

**Discussion by itsbhatt** [run-llama/LlamaIndexTS#993] [Q&A]
Hi, i am trying to integrate my local llamafile server with the llamaindex.ts but i couldn't find the doc for that i was the documentation of python version and it has the option in its documentation.

Can anyone please help here?

Thanks!

**Comment by marcusschiesser** [comment]
That's not supported yet. Supported LLMs are here: https://ts.llamaindex.ai/modules/llms/ and supported embeddings here: https://ts.llamaindex.ai/modules/embeddings/
